src/renderers/shaders/ShaderChunk/packing.glsl

```diff
@@ -30,10 +30,10 @@ float unpackRGBAToDepth( const in vec4 v ) {
 
 // NOTE: viewZ/eyeZ is < 0 when in front of the camera per OpenGL conventions
 
-float viewZToOrthoDepth( const in float viewZ, const in float near, const in float far ) {
+float viewZToOrthographicDepth( const in float viewZ, const in float near, const in float far ) {
   return ( viewZ + near ) / ( near - far );
 }
-float OrthoDepthToViewZ( const in float linearClipZ, const in float near, const in float far ) {
+float orthographicDepthToViewZ( const in float linearClipZ, const in float near, const in float far ) {
```

Comment:
It is not currently being used, but it is being used in future PRs (SAO effect, glossy mirror, etc.):

![image](https://cloud.githubusercontent.com/assets/588541/15619060/c2ceec60-241f-11e6-97ca-3d2813ef6226.png)

BTW glossy/rough mirror:

![image](https://cloud.githubusercontent.com/assets/588541/15619099/067ae392-2420-11e6-87af-bb9f5596eb24.png)